projects/cdk/schematics/ng-update/v4/steps/constants/identifiers-to-replace.ts

```diff
@@ -2419,4 +2419,110 @@ export const IDENTIFIERS_TO_REPLACE: ReplacementIdentifierMulti[] = [
             moduleSpecifier: '@taiga-ui/core',
         },
     },
+    {
+        from: {
+            name: 'WaResizeObserver',
+            moduleSpecifier: '@ng-web-api/resize-observer',
+        },
+        to: {
+            name: 'WaResizeObserver',
+            moduleSpecifier: '@ng-web-api/resize-observer',
+        },
+    },
+    {
+        from: {
+            name: 'IntersectionObserverModule',
+            moduleSpecifier: '@ng-web-api/intersection-observer',
+        },
+        to: {
+            name: 'WaIntersectionObserver',
+            moduleSpecifier: '@ng-web-api/intersection-observer',
```

Comment:
```suggestion
            moduleSpecifier: '@ng-web-apis/intersection-observer',
        },
        to: {
            name: 'WaIntersectionObserver',
            moduleSpecifier: '@ng-web-apis/intersection-observer',
```

(\+ all other similar mistakes)

Reply:
fixed